en/moderation.json

```diff
@@ -18,7 +18,7 @@
             "warn": "<@{userID}> do not send invites.",
             "log": {
                 "title": "Invite sent",
-                "invite": "Invite"
+                "invite": "Inv. Code" // Or Invite Code, inv. code would just make it smaller in my eyes. 
```

Comment:
"Invite Code" would probably be the best since users are a bit naïve. Remove the comment also and then it'll be 👌